pinot-core/src/main/java/org/apache/pinot/core/operator/query/DataSourceBasedAggregationOperator.java

```diff
@@ -56,64 +58,69 @@
  * For max value we use the last value from dictionary
  */
 @SuppressWarnings("rawtypes")
-public class DictionaryBasedAggregationOperator extends BaseOperator<IntermediateResultsBlock> {
+public class DataSourceBasedAggregationOperator extends BaseOperator<IntermediateResultsBlock> {
```

Comment:
`NonScanAggregationOperator` could be a better name because all indexes are part of the data source.
We should also update the javadoc for this class